trust-sdk/trust-sdk-specs.md

```diff
@@ -53,48 +78,62 @@ lrzEHpAAIAmoB0Yegc2ZqxtkSkXlYo_TEg4eBrjopGUj9ySxJh6JlfToGqR7yNKzV8cD_yN_jVR5YrVa
 7Tlfb_0eV47yG1Co&id=4
 ```
 
-### Sign Message
+### Sign simple transaction
+
+This command is the simplified version of `sdk_sign`, it accepts a simple `Transaction` object instead of heavier protobuf model
 
 #### Command
-* Command: `sdk_sign_message`
+
+* Command: `sdk_transaction`
 * Parameters:
   * `coin`: BIP44 coin code
-  * `data`: message to sign, caller must encode or (hash it) as hex string
-  * `meta`: dapp metadata. This attribute is optional and encoded as a dictionary. Check how it's encoded in [Dictionary Encoding](#dictionary-encoding).
+  * `to`: recipient address
+  * `amount`: amount in human-readable (unit) format
+  * `action`: flag that indicates if the wallet should broadcast the transaction. Valid values: `send`, `sign`.
+  * `token_id`: (Optional) token id, follows standard of unique identifier on the blockhain as smart contract address or asset ID
+  * `from`: (Optional) specifies which account/address to send
+  * `nonce`: (Optional) Custom nonce or sequence
+  * `fee_price`: (Optional) fee price in smallest unit
+  * `fee_limit`: (Optional) fee limit in smallest unit
+  * `meta`: (Optional) transaction data in hex format, Memo or Destination tag
 
 ##### Example
 
 ```shell
-sampleapp://sdk_sign_message?coin=60&data=4fe61e1a9fb1d18a78977ad1e9611e8c546d54743cf2ff1836fc6933df9f1a54&app=trustsdk&callback=sdk_sign_result&id=1
+trust://sdk_transaction?coin=60&to=0x1b38BC1D3a7B2a370425f70CedaCa8119ac24576&meta=0xa9059cbb0000000000000000000000000F36f148D6FdEaCD6c765F8f59D4074109E311f0c0000000000000000000000000000000000000000000000000000000000000001&token_id=token&nonce=447&fee_price=2112000000&fee_limit=21000&amount=0.001&action=send&callback=sdk_sign_result&id=1
 ```
```

Comment:
trust://sdk_transaction?action=transfer&asset=c1001_t0x514910771af9ca656af840dff83e8264ecf986ca&
to=0x514910771af9ca656af840dff83e8264ecf986ca&amount=0.01&callback=sample_app://sdk_sign_result&
confirm_type=sign&nonce=2&from=0xF36f148D6FdEaCD6c765F8f59D4074109E311f0c&meta=memo&fee_limit=1&
fee_price=1&id=1

Reply:
let's follow others, `callback=sample_app://sdk_sign_result` => `app=sample_app&callback=sdk_sign_result`

Reply:
why do we need to construct callback deeplink from parameters? We just build upon callback and open it.

Reply:
If I remember correctly, we previously had something like `sample_app://sdk_sign_result` in iOS url , `://` may cause failed to parse the url or percent encoded (which requires extra decode step)